roles/klaytn_bridge/inventory

```diff
@@ -1,5 +1,7 @@
 [ParentBridgeNode]
 PARENT0 ansible_host=1.2.3.4 ansible_user=PARENT_USER
+# You can define node-specific variables here e.g., parent_rpc_port
+PARENT1 ansible_host=1.2.3.5 ansible_user=PARENT_USER parent_rpc_port=1234
```

Comment:
It is good.
But it is a little hard to match parent with child by [item] index 0 and 1, etc.
Another grouping method use node number.

[ParentBridgeNode]
ParentNode01 ansible_host=1.2.3.4 node=node01 type=Parent rpc_port=1234
ParentNode02 ansible_host=1.2.3.5 node=node02 type =Parent  rpc_port=1234

[ChildBridgeNode]
ChildNode01 ansible_host=1.3.3.4 node=node01 type=Child rpc_port=1234
ChildNode02 ansible_host=1.3.3.5 node=node02 type =Child rpc_port=1234

You can find child bridge by [Child type =Child] + [node=node01]

Reply:
I agree current code, too.

Reply:
I will add an issue for a better design to enhance readability for the variables used in ansible.